draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -280,7 +280,7 @@ authority is required to administer them (although one format uses
 IEEE 802 node identifiers, others do not).  As a result, generation
 on demand can be completely automated, and used for a variety of
 purposes.  The UUID generation algorithm described here supports very
-high allocation rates of up to 10 million per second per machine if
+high allocation rates of up to 10 million per second per machine or more if
```

Comment:
With the bit space available it is possible to generate much more than 10 million identifiers per second so I think we should indicate that so people don't think it's somehow hard-limited to this.